tests/InlineTextTest.php

```diff
@@ -27,29 +24,53 @@ public static function expectedRenders()
                 new InlineText("  Hello world!  ", "¯\_(ツ)_/¯"),
                 "  ¯\_(ツ)_/¯Hello world!¯\_(ツ)_/¯  "
             ),
-            "expect only whitespace and subtype" => array(
+            "expect text with internal newlines" => array(
+                new InlineText("First\nSecond\n\n\nThird", ":"),
+                ":First:\n:Second:\n\n:Third:"
+            ),
```

Comment:
I'm assuming we strip newlines >2 and this is expected.

Reply:
@graycodes yep
https://github.com/conde-nast-international/copilot-markdown-generator-php/blob/master/tests/TextTest.php#L34